src/app/modules/card/card.component.scss

```diff
@@ -0,0 +1,12 @@
+.ukm-card{
+  width: 10vw;
+  height: 5vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 20px;
```

Comment:
I think we should place this margin-bottom in the parent component that uses this component.